modules/web/js/ballerina/configs/designer-defaults.js

```diff
@@ -263,4 +264,11 @@ export const panelHeading = {
         heading: 14,
         padding: 3
     }
-};
\ No newline at end of file
+};
+
+export const timeout = {
```

Comment:
Can this be renamed to a more generic name ? then we can use the same config in another statement type if required.